channeldb/migration13/migration_test.go

```diff
@@ -0,0 +1,112 @@
+package migration13
+
+import (
+	"testing"
+
+	"github.com/coreos/bbolt"
+	"github.com/lightningnetwork/lnd/channeldb/migtest"
+)
+
+var (
+	zeroTime             = "0000000000000000"
+	noFailureMessage     = "00"
+	failureReasonUnknown = "00"
+	zeroFailureSourceIdx = "00000000"
+
+	hash1         = "02acee76ebd53d00824410cf6adecad4f50334dac702bd5a2d3ba01b91709f0e"
```

Comment:
looks like the test data can be set in a table, and then create the pre- and post-maps programmatically from iterating this table? Or maybe that would defeat the purpose of spelling out the db in the first place?

Reply:
It could, but it wouldn't be very different. Also not all test payments have the same fields.